examples/drake_walking.ipynb

```diff
@@ -72,11 +72,11 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 5,
    "metadata": {},
```

Comment:
Why did this file change ?

Reply:
in that notebook I added a a code snippet to set the feet corners, and I think I committed some metadata with it. Since this is quite unrelated with this PR I think I will remove this file from the PR changes.

Reply:
Done with rebase